packages/broker/test/integration/plugins/operator/InspectRandomNodeHelper.test.ts

```diff
@@ -0,0 +1,167 @@
+const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '10.200.10.1'}:8000/subgraphs/name/streamr-dev/network-subgraphs`
+
+const logger = new Logger(module)
+
+jest.setTimeout(600 * 1000)
+
+describe("InspectRandomNodeHelper", () => {
+    const chainURL = config.rpcEndpoints[0].url
+
+    let provider: Provider
+    let token: TestToken
+    let adminWallet: Wallet
+    let streamId1: string
+    let streamId2: string
+    let graphClient: TheGraphClient
+
+    beforeAll(async () => {
+        provider = new JsonRpcProvider(chainURL)
+        logger.debug("Connected to: ", await provider.getNetwork())
+
+        const streamCreatorKey = "0xfe1d528b7e204a5bdfb7668a1ed3adfee45b4b96960a175c9ef0ad16dd58d728"
+        adminWallet = new Wallet(streamCreatorKey, provider)
+
+        token = new Contract(config.contracts.LINK, tokenABI) as unknown as TestToken
+        const timeString = (new Date()).getTime().toString()
+        const streamPath1 = "/operatorclienttest-1-" + timeString
+        const streamPath2 = "/operatorclienttest-2-" + timeString
+        streamId1 = adminWallet.address.toLowerCase() + streamPath1
+        streamId2 = adminWallet.address.toLowerCase() + streamPath2
+        const streamRegistry = new Contract(config.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        logger.debug(`creating stream with streamId1 ${streamId1}`)
+        await (await streamRegistry.createStream(streamPath1, "metadata")).wait()
+        logger.debug(`creating stream with streamId2 ${streamId2}`)
+        await (await streamRegistry.createStream(streamPath2, "metadata")).wait()
+
+        graphClient = new TheGraphClient({
+            serverUrl: theGraphUrl,
+            fetch,
+            logger: logger
+        })
+    })
+
+    describe("InspectRandomNodeHelper methods", () => {
+        it("getSponsorshipsOfOperator, getOperatorsInSponsorship", async () => {
+            const { operatorWallet, operatorContract, operatorConfig } = await createWalletAndDeployOperator(
+                provider, config, theGraphUrl
+            )
+            const inspectRandomNodeHelper = new InspectRandomNodeHelper(operatorConfig)
+            
+            logger.debug("Added OperatorClient listeners, deploying Sponsorship contract...")
+            const sponsorship = await deploySponsorship(config, operatorWallet, {
+                streamId: streamId1 })
+            const sponsorship2 = await deploySponsorship(config, operatorWallet, {
+                streamId: streamId2
+            })
+
+            logger.debug(`Sponsorship deployed at ${sponsorship.address}, delegating...`)
+            await (await token.connect(operatorWallet).transferAndCall(operatorContract.address, parseEther("200"), operatorWallet.address)).wait()
+
+            logger.debug("Staking to sponsorship...")
+            await (await operatorContract.stake(sponsorship.address, parseEther("100"))).wait()
```

Comment:
Could use constant here instead of magic number? E.g. STAKE1 = 123, STAKE2 = 234 and at line then STAKE1+STAKE2

Reply:
the test does not test any logic, like "+"